### PR TITLE
Update jupyter ui to allow creating notebook in all available namespces

### LINKED ISSUE #3000: Jupyter UI should allow creating notebooks in all namespaces
in Kubeflow v 0.5, UI allows to launch notebook in only Kubeflow namespace [#2925](https://github.com/kubeflow/kubeflow/issues/2925). 
This is because, Kubeflow 0.5 i) by default GCP and K8S service accounts are only available in kubeflow namespace, and ii) every notebook is expected to apply an admission hook to mount GCP and K8S service accounts.  Therefore, notebooks fails to create in other namespaces, and consequently UI disables launching notebooks in other namespaces.

We need to modify UI so that it allows launching notebooks in all namespaces. This is because we would like  to enable testing various works for multi-user Jupyter. Further, if users manually create gcp and and k8s service accounts in their desired namespaces, [as described here,](https://github.com/zabbasi/website/blob/jupyter2/content/docs/components/jupyter.md) they can safely create notebook.

### PR DESCRIPTION
**Pull request by zabbasi**
Undo #2951
Fix #3000 
Needs testing
To unblock #3022 

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/kubeflow/3074)

//cc @kimwnasptd @avdaredevil

### CODE REVIEW

File: components/jupyter-web-app/kubeflow_jupyter/default/templates/notebooks.html [hunk 1]
```diff
@@ -13,7 +13,7 @@
     <select class="mdl-textfield__input" id="ns-select" name="ns">
       <!-- Fill with the namespaces the user can see -->
       {% for ns in namespaces %}
-        <option value="{{ ns }}">{{ ns }}</option>
+        <option value="{{ ns }}" {{ ns == 'kubeflow'|yesno:"selected" }}>{{ ns }}</option>
```
Comment (avdaredevil) [inline components/jupyter-web-app/kubeflow_jupyter/default/templates/notebooks.html]: This cherry-pick is from a PR that had a syntax error.


You can use:
```suggestion
        <option value="{{ ns }}" {% if ns == 'kubeflow' %}selected{% endif %}>{{ ns }}</option>
```

Comment (zabbasi) [reply components/jupyter-web-app/kubeflow_jupyter/default/templates/notebooks.html]: 👍

**Comment by zabbasi** [comment]
duplicated to #3019

**Pull request closed without merging**